daprdocs/content/en/concepts/overview.md

```diff
@@ -43,84 +43,78 @@ Each of these building blocks is independent, meaning that you can use one, some
 
 ## Sidecar architecture
 
-Dapr exposes its APIs as a sidecar architecture, either as a container or as a process, not requiring the application code to include any Dapr runtime code. This makes integration with Dapr easy from other runtimes, as well as providing separation of the application logic for improved supportability.
+Dapr exposes its HTTP and gRPC APIs as a sidecar architecture, either as a container or as a process, not requiring the application code to include any Dapr runtime code. This makes integration with Dapr easy from other runtimes, as well as providing separation of the application logic for improved supportability.
 
-## Hosting Environments
-Dapr can be hosted in multiple environments, including self hosted for local development or to deploy to a group of VMs, Kubernetes and edge environments such as Azure IoT Edge.
+<img src="/images/overview-sidecar-model.png" width=700>
 
-### Self hosted
+## Hosting environments
 
-In self hosted mode Dapr runs as a separate side-car process which your service code can call via HTTP or gRPC. In self hosted mode, you can  also deploy Dapr onto a set of VMs.
+Dapr can be hosted in multiple environments, including self-hosted for local development and Kubernetes for production workloads.
```

Comment:
Self-hosted is not just about local development. With the introduction of Consul, the VM scenario is now possible. You may not want to change this here, but it should be made clear about VMs being a hosting option.

Reply:
I'll update